src/docs/product/data-management-settings/server-side-sampling/index.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Server-side Sampling
+sidebar_order: 1
+description: "Learn about how you can control which transactions Sentry retains."
+---
+
+Server-side sampling lets you control what transactions Sentry retains by setting sampling rules and rates so you see more of the transactions you want to explore further, and less of the ones you don’t, without reconfiguring the SDK or redeploying your application.
```

Comment:
I think we should make the connection to client-side and explicitly explain the difference to client-side sampling in the end. feel free to tweak for grammar and language.

Feel free to tweak.
```suggestion
Server-side sampling lets you control what transactions Sentry retains by setting sampling rules and rates so you see more of the transactions you want to explore further, and less of the ones you don’t. In contrast to client-side sampling [LINK], server-side sampling lets you manage your sampling configuration centrally and lets you make changes without the need to reconfigure the SDK and to redeploy your application.
```